src/lib/docs/AccountKit/LinkAuthMethod.md

```diff
@@ -0,0 +1,30 @@
+---
+title: LinkAuth (Method)
+description: Used to generate an action that creates a dependency between a smart contract action and a permission level.
+category: AccountKit
+published: true
+order: 9
+---
+
+# Linkauth
+
+The `linkAuth` method provides a way for an Antelope account owner to require a specific account permission level when executing a specific action on a smart contract.
```

Comment:
"provides a way for an Antelope account to link/limit specific permissions to specific smart contract actions"

Reply:
Ahhhh I thought it was the other way around 😅